mkdocs/docs/getting-started/first-experiment.md

```diff
@@ -4,16 +4,16 @@ template: main.html
 
 # Your First Experiment
 
-!!! tip "Scenario: Validate service-level objectives (SLOs) for a K8s app"
-    **Problem**: You have a K8s app. You want to verify that it satisfies latency and error rate SLOs.
+!!! tip "Scenario: Validate service-level objectives (SLOs) for a Kubernetes app"
+    **Problem**: You have a Kubernetes app. You want to verify that it satisfies latency and error rate SLOs.
 
-    **Solution**: In this tutorial, you will launch a K8s app along with an Iter8 experiment. Iter8 will [validate that the new satisfies latency and error-based objectives (SLOs)](../concepts/buildingblocks.md#slo-validation) using [built-in metrics](../metrics/builtin.md).
+    **Solution**: In this tutorial, you will launch a Kubernetes app along with an Iter8 experiment. Iter8 will [validate that the app satisfies latency and error-based objectives (SLOs)](../concepts/buildingblocks.md#slo-validation) using [built-in metrics](../metrics/builtin.md). During this validation, Iter8 will generate HTTP GET requests for the app.
 
     ![SLO Validation](../images/slo-validation.png)
     
-??? warning "Setup K8s cluster and local environment"
-    1. Setup [K8s cluster](setup-for-tutorials.md#local-kubernetes-cluster)
-    2. [Install Iter8 in K8s cluster](install.md)
+??? warning "Setup Kubernetes cluster and local environment"
+    1. Setup [Kubernetes cluster](setup-for-tutorials.md#local-kubernetes-cluster)
+    2. [Install Iter8 in Kubernetes cluster](install.md)
     3. Get [Helm 3.4+](https://helm.sh/docs/intro/install/).
     4. Get [`iter8ctl`](install.md#get-iter8ctl)
     5. Fork the [Iter8 GitHub repo](https://github.com/iter8-tools/iter8). Clone your fork, and set the ITER8 environment variable as follows.
```

Comment:
`set the ITER8 environment variable`

I think `ITER8` should have code syntax. I think there may be other instances of this in the docs.

Reply:
Fixed.